### PR TITLE
T6231: Add documentation for building Mellanox OFED drivers

### PR DESCRIPTION
**Pull request by sempervictus**
## Change Summary
Docs for building OFED

## Related Task(s)
[T6231](https://vyos.dev/T6231)

## Related PR(s)
https://github.com/vyos/vyos-build/pull/665

## Backport
N/A

## Checklist:

- [ ] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-documentation/blob/current/CONTRIBUTING.md) document

### CODE REVIEW

File: docs/contributing/build-vyos.rst [hunk 1]
```diff
@@ -741,6 +741,38 @@ binaries in ``vyos-build/packages/linux-kernel`` from which you can copy them
 to the ``vyos-build/packages`` folder for inclusion during the ISO build.
 
 
+Mellanox OFED
+^^^^^^^^^
```
Comment (rebortg) [inline docs/contributing/build-vyos.rst]: please add some `^` to have the same length as the headline text

**Review by rebortg — CHANGES_REQUESTED** [review]

File: docs/contributing/build-vyos.rst (hunk 1 above)
Comment (dmbaturin) [reply docs/contributing/build-vyos.rst]: @sempervictus Could you make the tiny fix in question?

Comment (sempervictus) [reply docs/contributing/build-vyos.rst]: Thanks for pinging, didn't notice this in the mail flood. Yessir, will get that done by weekend (on the move presently)

Comment (dmbaturin) [inline docs/contributing/build-vyos.rst]:
```suggestion
^^^^^^^^^^^^
```

File: docs/contributing/build-vyos.rst [hunk 2]
```diff
@@ -741,6 +741,38 @@ binaries in ``vyos-build/packages/linux-kernel`` from which you can copy them
 to the ``vyos-build/packages`` folder for inclusion during the ISO build.
 
 
+Mellanox OFED
+^^^^^^^^^^^^
```
Comment (dmbaturin) [inline docs/contributing/build-vyos.rst]:
```suggestion
^^^^^^^^^^^^^
```

**Review by c-po — APPROVED** [review]

**Pull request merged**